examples/ps/main.py

```diff
@@ -0,0 +1,4 @@
+from ollama import ps
+
+response = ps()
+print(response)
```

Comment:
This example should be a bit more involved. To really show ps, you should load a model by calling chat or generate